SingularityUI/app/actions/api/history.es6

```diff
@@ -37,6 +37,22 @@ export const FetchDeployForRequest = buildApiAction(
   })
 );
 
+export const FetchTaskSearchParams = buildApiAction(
+  'FETCH_TASK_HISTORY',
+  ({requestId = null, deployId = null, host = null, lastTaskStatus = null, startedAfter = null, startedBefore = null, orderDirection = null, count, page}) => {
+    let params = [];
+    if (requestId) params.push(`&requestId=${requestId}`);
+    if (deployId) params.push(`&deployId=${deployId}`);
+    if (host) params.push(`&host=${host}`);
+    if (lastTaskStatus) params.push(`&lastTaskStatus=${lastTaskStatus}`);
+    if (startedAfter) params.push(`&startedAfter=${startedAfter}`);
+    if (startedBefore) params.push(`&startedBefore=${startedBefore}`);
+    if (orderDirection) params.push(`&orderDirection=${orderDirection}`);
+    return {
+      url: `/history/tasks?count=${count}&page=${page}${params.join('')}`
+    };
```

Comment:
we may want to use a function for building query params (like `$.param()`, but I'd rather not depend on jQuery), since it will handle escaping, etc. for us